openquake/ghm/mosaic.py

```diff
@@ -39,6 +39,22 @@
            'nea': {'RUS': ['75 0 180 0 180 88 75 88',
                            '-180 0 -160 0 -160 88 -180 88']},
            'nwa': {'RUS': ['75 0 25 0 25 10 25 20 25 40 25 50 25 88 75 88']},
+           'oan': {'ATF': ['49.5 -50.1 71 -50.1 71 -45.6 49.5 -45.6']},
```

Comment:
do we have a model 'oan'? (Not sure if it matters code-wise... )

Reply:
whoops this should be 'pan' (polar Antarctic)